analyses/fusion-summary/01-fusion-summary.Rmd

```diff
@@ -203,9 +257,104 @@ allFuseEwing %>%
   write_tsv(ewingsFile)
 ```
 
+#### Perform selection for LGAT fusions
```

Comment:
Maybe we can have smaller chunks per selection criteria for LGAT below or do we want it 1 chunk per broad_histology ?
 - Added all 3' kinase fusions which are in-frame and retain the kinase domain
 - For 5' kinase fusions, added those which are in-frame and retain the kinase domain
- For 5' kinase fusions that don't meet 3., check whether it has a reciprocal fusion and whether that reciprocal is in-frame and the kinase domain is retained. Then, add those 5' kinase fusions.

Reply:
updated to smaller chunks

Reply:
>I wonder if the CI is errors out with the test data?

This may be due to fusions not being present in the test data. @jaclyn-taroni I think we need to add more fusions to the test data. I only see the below fusions in the test fusion file, so we would have zeroed out dataframes in this module, which may be what is causing the CI failure.

```
intersect(subsetFuseLGAT[,c("Sample", "FusionName")], test.fus[,c("Sample", "FusionName")])
# A tibble: 4 x 2
  Sample      FusionName    
  <chr>       <chr>         
1 BS_49852ZBH KIAA1549--BRAF
2 BS_7P83Z28P KIAA1549--BRAF
3 BS_BMMY0H73 NTRK2--NTRK2  
4 BS_F6V1Y4QS GOPC--ROS1  
```

Reply:
Yes, we already have special consideration for fusions in how we generate the test data. It is a matter of updating those steps to reflect your changes here.

See the docs for [`create-subset-files`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/b39ff2e25439a6c66a4f278a5314984da3eb8eb0/analyses/create-subset-files/README.md#special-considerations) and the code where we make that accommodation:

https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/b39ff2e25439a6c66a4f278a5314984da3eb8eb0/analyses/create-subset-files/02-subset_files.R#L112

If you want to file a new pull request with the required changes, we can probably regenerate the files and get them uploaded to the right spot before this gets merged.

Reply:
@jaclyn-taroni quick question - I have updated the fusions/genes to select. Do you just want the change in `OpenPBTA-analysis/analyses/create-subset-files/02-subset_files.R` or do you want me to also do the subsetting of files? Sounds like you would do this on your end and upload to s3?

Reply:
Nevermind, I did a quick subset of the putative oncogenic and tested it, and it works. PR coming in a min!